troubadix/plugins/valid_oid.py

```diff
@@ -377,6 +405,47 @@ def check_content(
                     return
 
                 return
+        if "1.3.6.1.4.1.25623.1.3." in oid:
+            family_pattern = get_special_script_tag_pattern(
+                SpecialScriptTag.FAMILY
+            )
+            family_match = family_pattern.search(file_content)
+            if not family_match or not family_match.group("value"):
+                yield LinterError(
+                    "VT is missing a script name!",
```

Comment:
Two notes:
1. This is actually checking for a "script family" and not for a "script name"
2. As the `SpecialScriptTag.FAMILY` is now called /evaluated two times in this plugin would it make sense to move it at the top (e.g. directly after the `if "1.3.6.1.4.1.25623.1." not in oid:` check)?
  - a VT without a `script_family()` tag is invalid anyway
  - Optional: Move the `SpecialScriptTag.NAME` check to the top as well